SRTNet.cpp

```diff
@@ -338,47 +403,30 @@ bool SRTNet::startClient(const std::string& host,
         return false;
     }
 
-    // Get all remote addresses for connection
-    struct addrinfo hints = {};
-    struct addrinfo* resolvedAddresses;
-    hints.ai_socktype = SOCK_DGRAM;
-    hints.ai_protocol = IPPROTO_UDP;
-    hints.ai_family = AF_UNSPEC;
-    std::stringstream portAsString;
-    portAsString << port;
-    int result = getaddrinfo(host.c_str(), portAsString.str().c_str(), &hints, &resolvedAddresses);
-    if (result) {
-        SRT_LOGGER(true, LOGG_FATAL,
-                   "Failed getting the IP target for > " << host << ":" << port << " Errno: " << result);
+    // Try to connect to the server
+    ClientConnectStatus status = clientConnectToServer();
+    if (status == failToResolveAddress) {
+        SRT_LOGGER(true, LOGG_FATAL, "Failed to resolve address for " <<
```

Comment:
Should this be ERROR rather than FATAL. FATAL for me is an issue where the operation of the process cannot continue and you have to close it. Golang even terminates the process if you call log with fatal level.

Reply:
Same on other locations where FATAL is used which I think should be ERROR

Reply:
Yeah, sounds good, I've updated that now.